scripts/release_info.py

```diff
@@ -409,7 +642,83 @@ def main():
             for link in issue_links:
                 out.write('* [%s](%s)  \n' % (link, link))
 
-    python_utils.PRINT('Done. Summary file generated in ../release_summary.md')
+    python_utils.PRINT('Done. Summary file generated in %s' % (
+        RELEASE_SUMMARY_FILEPATH))
+
+    while True:
+        python_utils.PRINT(
+            '******************************************************')
+        python_utils.PRINT(
+            'Please update %s to:\n'
+            '- have a correct changelog for '
+            'updating the CHANGELOG file\n'
```

Comment:
You can probably move some of this to the previous line?

Reply:
Done.